projects/angular/src/forms/styles/_password.clarity.scss

```
@@ -16,7 +16,7 @@
 
   & + button.clr-input-group-icon-action[disabled] {
     .clr-password-eye-icon {
-      color: forms-variables.$clr-forms-password-disabled-eye-icon-color;
+      fill: forms-variables.$clr-forms-password-disabled-eye-icon-color;
```

```suggestion
      --color: forms-variables.$clr-forms-password-disabled-eye-icon-color;
```

I think we should use the `cds-icon` CSS custom property API for this.

---

We use `fill` everywhere with the exception of `responsive-nav`.
I don't say it is correct, but more about consitancy in our code.

---

Either way is fine for this PR, but I think we should open a follow-up `chore` PR to fix it everywhere.

---

@mivaylo please open a separate chore PR for this comment.

---

@valentin-mladenov , @kevinbuhmann , opened this one: https://github.com/vmware-clarity/ng-clarity/pull/1504